spec/models/ability_for_event_organizer_spec.rb

```diff
@@ -0,0 +1,258 @@
+require 'spec_helper'
+require 'cancan/matchers'
+
+describe 'Event_organizer' do
+  fixtures :all
+
+  subject { Ability.new(user) }
+  let(:user) { users(:josh) }
```

Comment:
Minor thing... I would write this in one line. There is no other code here that would use the user variable.